.github/workflows/aks.yaml

```diff
@@ -230,3 +230,12 @@ jobs:
           description: Connectivity test cancelled
           state: pending
           target_url: ${{ env.check_url }}
+
+      - name: Send slack notification
+        if: ${{ (cancelled() || failure()) && (github.event_name == 'schedule' || github.event_name == 'push') }}
```

Comment:
In cilium-cli, this was changed to `!success()`. Should we also make that consistent here as well? https://github.com/cilium/cilium-cli/pull/237#discussion_r644024948

Reply:
:shipit: